cesium_app/handlers/model.py

```diff
@@ -102,14 +105,17 @@ def post(self):
             featureset=fset_data, model_type=model_type,
             model_parameters=model_params,
             params_to_optimize=params_to_optimize)
-        future = executor.submit(joblib.dump, computed_model, model_file.uri)
-        closed = executor.submit(xr.Dataset.close, fset_data)
+        score_future = executor.submit(build_model.score_model, computed_model,
```

Comment:
@stefanv these changes were our attempt at getting ensure that all the steps are always executed without any race conditions; can you think of a better way?